Gifski/ConversionViewController.swift

```diff
@@ -42,16 +42,16 @@ final class ConversionViewController: NSViewController {
 		view = wrapper
 	}
 
-	override func viewDidLoad() {
-		super.viewDidLoad()
+	override func viewDidAppear() {
+		super.viewDidAppear()
+
+		view.window?.makeFirstResponder(self)
 
 		start(conversion: conversion)
 	}
 
 	/// Gets called when the Esc key is pressed.
-	/// Reference: https://stackoverflow.com/a/42440020
-	@objc
-	func cancel(_ sender: Any?) {
+	override func cancelOperation(_ sender: Any?) {
```

Comment:
I also changed this as with the macOS 10.14 SDK, NSViewController has `cancelOperation` too.